Sources/BraintreeThreeDSecure/BTPaymentFlowDriver+ThreeDSecure.m

```diff
@@ -83,14 +95,6 @@ - (void)performThreeDSecureLookup:(BTThreeDSecureRequest *)request
             requestParameters[@"additionalInfo"] = additionalInformation;
         }
 
-        if (request.challengeRequested) {
```

Comment:
I moved these up so that everything that isn't part of `additionalInformation` is grouped together.